server/game/cards/14.4-CoC/StoicRival.js

```diff
@@ -0,0 +1,20 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes } = require('../../Constants');
+
+class StoicRival extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Dishonor a participating character with fewer attachments',
+            condition: context => context.source.attachments.size() > 0 && context.source.isParticipating(),
+            target: {
+                cardType: CardTypes.Character,
+                cardCondition: (card,context) => card.isParticipating() && card.attachments.size() <context.source.attachments.size(),
+                gameAction: ability.actions.dishonor()            
+			}
```

Comment:
Don't accept the ability argument but import and use the AbilityDsl, the way you did it is out of style.

Reply:
Fixed in commit 97a811c...

I based the card on core set hence the outdated style. Linted Stoic Rival too, will update tests next.